common/transformations/camera.py

```diff
@@ -6,8 +6,10 @@
 ## -- hardcoded hardware params --
 eon_f_focal_length = 910.0
 eon_d_focal_length = 650.0
+# tici_f: narrow road cam. tici_e: wide road cam
 tici_f_focal_length = 2648.0
-tici_e_focal_length = tici_d_focal_length = 567.0 # probably wrong? magnification is not consistent across frame
+tici_d_and_e_focal_length = 567.0 # probably wrong? magnification is not consistent across frame
+tici_e_focal_length = tici_d_focal_length = tici_d_and_e_focal_length
```

Comment:
Unsure if we want to keep tici_e_focal_length since it isn't used anywhere.

Reply:
it's used in an internal repo